test/unit/getOverlayEntry.test.js

```diff
@@ -0,0 +1,78 @@
+const getOverlayEntry = require('../../lib/utils/getOverlayEntry');
+
+const ErrorOverlayEntry = require.resolve('../../client/ErrorOverlayEntry');
+const ReactRefreshEntry = require.resolve('../../client/ReactRefreshEntry');
+
+const DEFAULT_OPTIONS = {
+  overlay: {
+    entry: ErrorOverlayEntry,
+  },
+};
+
+describe('getOverlayEntry', () => {
+  it('should append legacy WDS entry when required', () => {
```

Comment:
Can we also add a test for the baseline case (i.e. pass in default comes out as default)?